airflow/www/views.py

```diff
@@ -1415,6 +1416,9 @@ def graph(self, session=None):
             flash('DAG "{0}" seems to be missing.'.format(dag_id), "error")
             return redirect('/admin/')
 
+        dt_nr_dr_data = get_date_time_num_runs_dag_runs_form_data(request, session, dag)
```

Comment:
This variable name makes me cry

Reply:
Don't get me started on the function name

Reply:
I feel guilty because both were introduced by me in https://github.com/apache/incubator-airflow/pull/3450